R/count.R

```diff
@@ -128,7 +129,11 @@ prep.step_count <- function(x, training, info = NULL, ...) {
   check_type(training[, col_name], types = c("string", "factor", "ordered"))
 
   if (length(col_name) > 1) {
-    rlang::abort("The selector should select at most a single variable")
+    cli::cli_abort(c(
+      x = "The selector should select at most a single variable.",
+      i = "The following {length(col_name)} were selected:\\
```

Comment:
```suggestion
      i = "The following {length(col_name)} were selected: \\
```

Reply:
In some cases, you write 

```
! = ...the following:
"*" = x and y
```

and in others

```
the following: x and y
```

My preference is probably the latter, but I'd say either as fine as long as a given package is internally consistent.